api/src/main/java/org/openmrs/module/cfl/api/service/messages/impl/MultiProjectDefaultPatientTemplateServiceImpl.java

```diff
@@ -53,7 +63,12 @@ private List<PatientTemplate> getProjectBasedPatientTemplatesWithDefaultValues(P
         new ArrayList<>(templates.size() + templates.size() * actors.size());
 
     for (Template template : templates) {
-      patientTemplates.add((new PatientTemplateBuilder(template, patient)).build());
+      PatientTemplate patientTemplate =
+          Context.getService(PatientTemplateService.class)
```

Comment:
I would move the getService to the outside for - not sure how much performance we'll get, but it would look like a correct approach.

Reply:
right, improved